src/broker.ts

```diff
@@ -86,7 +86,7 @@ export const startBroker = async (config: Todo) => {
         port: config.network.port,
         id: brokerAddress,
         name: networkNodeName,
-        trackers,
+        trackers: trackers as string[],
```

Comment:
Is this explicit cast necessary? Above we declare trackers as `let trackers: string[]`.